solr/core/src/test/org/apache/solr/handler/component/DistributedExpandComponentTest.java

```diff
@@ -153,6 +153,23 @@ public void test() throws Exception {
     assertExpandGroupCountAndOrder("group3", 1, results, "9");
     assertExpandGroupCountAndOrder("group4", 1, results, "14");
 
+    //Test expand.rows = 0 - no docs only expand count
+
+    params = new ModifiableSolrParams();
```

Comment:
Is it possible to add a case w/o collapse too (with expand.field) and with fl with and w/o `score`?

Reply:
done